app/src/pages/inside/launchesPage/modals/launchAnalysisModal/launchAnalysisModal.jsx

```diff
@@ -1,22 +1,24 @@
 import { Component } from 'react';
 import track from 'react-tracking';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
 import { injectIntl, intlShape, defineMessages } from 'react-intl';
 import { withModal, ModalLayout } from 'components/main/modal';
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
 import { LAUNCHES_MODAL_EVENTS } from 'components/main/analytics/events';
 import { InputRadio } from 'components/inputs/inputRadio';
 import { InputCheckbox } from 'components/inputs/inputCheckbox';
 import { LAUNCH_ANALYZE_TYPES } from 'common/constants/launchAnalyzeTypes';
-import { activeProjectSelector } from 'controllers/user';
 import classNames from 'classnames/bind';
 import styles from './launchAnalysisModal.scss';
 
 const cx = classNames.bind(styles);
 
 const messages = defineMessages({
-  analyseButton: {
+  MODAL_TITLE: {
+    id: 'analysisItemsModal.title',
+    defaultMessage: 'ANALYSE LAUNCHES',
```

Comment:
In `ModalLayout/ModalHeader` we have `text-transform: uppercase` css property for modal titles.
So please remove uppercase here